plotsquared/why-plotsquared.md

```diff
@@ -0,0 +1,53 @@
+# Why should you switch to PlotSquared?
+
+## Speed
+
+* Async: clear plots more than 800 times faster than PlotMe
+* Plot auto doesn't lag or crash the server
+* Much faster chunk generation, plot moving, swapping + everything
+* Supports huge plot sizes (e.g. 1024x1024 of custom terrain)
+* Works with hundreds of thousands of plots
+
+## Reduce lag
+
+* Plot purging and old plot removal
+* Optional dynamic world border and world chunk trimming
+* Mass plot condensation
+* Redstone clock prevention
+* Mob limiting
+
+## Safety
+
+* You can safely allow redstone, tnt without worrying about lag, griefing from sand cannons, pistons, tnt, ...
+* You have a reliable [WorldEdit](https://dev.bukkit.org/projects/worldedit) / [FastAsyncWorldEdit](https://www.spigotmc.org/resources/fast-async-worldedit.13932) restriction
+* Chunk and WorldEdit processor to prevent lag, crashes and data corruption
+* Lots of redundancies. e.g. Backup your database with `/plot database`, or if you loose both, you can still recover based on plot sign information.
+
+## Looks
+
+* Add amazing roads schematics and plot schematics to existing worlds
+* Easily change the message color scheme
+* Titles (entry notifications)
+* Per plot time / weather / music via [plot flags](/plotsqured/plot-flags)
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.